core/src/raw/oio/write/api.rs

```diff
@@ -30,8 +30,8 @@ use crate::*;
 pub enum WriteOperation {
     /// Operation for [`Write::write`]
     Write,
-    /// Operation for [`Write::sink`]
-    Sink,
```

Comment:
Why change name?

Reply:
pipe is more widely with between writers and readers.